prometheus.go

```diff
@@ -3,35 +3,44 @@ package aperture
 import (
 	"fmt"
 	"net/http"
+	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-const streamIDLabel = "streamID"
-
 var (
 	// mailboxCount tracks the current number of active mailboxes.
 	mailboxCount = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace: "hashmail",
 		Name:      "mailbox_count",
 	})
 
-	// mailboxReadCount counts each time a mailbox pair is being used.
-	// A session consists of a bidirectional stream each using a mailbox
-	// with an ID that overlaps for the first 63 bytes and differ for the
-	// last bit. So in order to obtain accurate data about a specific
-	// mailbox session, the stream ID that will be recorded is the first
-	// 16 bytes of the session ID and we will only record the odd stream's
-	// reads so that we don't duplicate the data.
-	mailboxReadCount = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Namespace: "hashmail",
-			Name:      "mailbox_read_count",
-		}, []string{streamIDLabel},
-	)
+	// activeSessions tracks the active session count for mailbox
```

Comment:
nit: please finish all sentences with punctuation.

Reply:
Will fix this.